webpack.common.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-undef*/
```

Comment:
This is here for a reason - so VS code doesn't report errors where there are none.